src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -393,7 +377,6 @@ public void widgetDefaultSelected(SelectionEvent e)
         dt.setTransfer(new Transfer[]{ TextTransfer.getInstance() });
         dt.addDropListener(new DropTargetAdapter()
         {
-            @Override
             public void drop(DropTargetEvent event)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)